gate-web/src/test/java/com/netflix/spinnaker/gate/security/oauth/OAuth2Test.java

```diff
@@ -17,14 +17,17 @@
 package com.netflix.spinnaker.gate.security.oauth;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.oauth2Login;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.http.HttpHeaders;
+import org.springframework.security.test.context.support.WithMockUser;
 import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
```

Comment:
Not part of this PR, but the tests pass for me without
```
"retrofit.enabled=true",
```
below.  They also pass if I set
```
"retrofit.enabled=false",
```
Given that we've migrated to retrofit2, can you please remove this line?

Reply:
https://github.com/spinnaker/gate/pull/1874/commits/233e3ea0c2b80abd98864a6aa6257e703f1fa3fe